nautobot/docs/user-guide/core-data-model/overview/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Understanding the data model is key to effectively navigating Nautobot. The number of models in Nautobot Core alone is nearly 200 models which does not include any of the models that come from Nautobot apps. This introduction does not attempt to document all of models, but instead intends to highlight the most common models to be used as a quick reference.
+
+## Fundamental Model Simplified
+
+The models included are generally the most used models and the most used attributes of those models, with the following points of consideration:
+
+- Tenants are an often used model, however they connect to most models and in the interest of simplicity not included.
```

Comment:
I am not sure what this sentence is trying to say. Maybe?
```suggestion
- Tenants are an often used model. However, they connect to most models and are not included in the interest of simplicity.
```

Reply:
I think the point here is that the Tenant model interconnects to most of these models but it's being intentionally omitted from the diagram to keep it more readable.

Reply:
@glennmatthews was correct, going to keep the original for now.